docker/internal-auth/env.template

```diff
@@ -98,3 +98,9 @@ FIFTYONE_APP_ENABLE_QUERY_PERFORMANCE=true
 # This should not exceed the value set in the deployment's license file for max concurrent delegated operators.
 # Set to 3 by default, meaning delegated operations can be executed without an external executor system.
 FIFTYONE_DELEGATED_OPERATOR_WORKER_REPLICAS=3
+
+# Global banner text to be displayed on top of all pages
+FIFTYONE_APP_BANNER_TEXT=
+
+# Global banner background color to be displayed on top of all pages
+FIFTYONE_APP_BANNER_COLOR=
```

Comment:
I would not add these here; document them in the README

it feels like _most_ customers won't need/use this...